upgrading/automated_upgrades.adoc

```diff
@@ -254,7 +248,7 @@ scenarios:
 
 - Upgrading existing {product-title} 3.6 clusters to 3.7
 - Upgrading existing {product-title} 3.7 clusters to the latest
-xref:../../release_notes/ocp_3_7_release_notes.html#ocp-37-asynchronous-errata-updates[asynchronous
+xref:../release_notes/ocp_3_7_release_notes.html#ocp-37-asynchronous-errata-updates[asynchronous
```

Comment:
This asynchronous errata updates link is broken in the preview build

Reply:
Fixt